source/routing/tests/routing_api_test.cpp

```diff
@@ -3086,14 +3086,21 @@ nr::NightBusFilter::Params get_default_filter_params() {
                                       nr::NightBusFilter::default_max_factor, nr::NightBusFilter::default_base_factor};
 }
 
-std::tuple<nr::Journey, nr::Journey> build_night_bus_journeys(ed::builder& b) {
-    b.vj("vj1")("A1", "09:00"_t, "09:00"_t)("C1", "12:00"_t, "12:00"_t);
-
+std::tuple<nr::Journey, nr::Journey, nr::Journey> build_night_bus_journeys(ed::builder& b) {
+    b.vj("vj0")("A0", "08:55"_t, "08:55"_t)("C1", "23:30"_t, "23:30"_t);  // sooner departure, but worst journey
+    b.vj("vj1")("A1", "09:00"_t, "09:00"_t)("C1", "12:00"_t, "12:00"_t);  // best journey clockwise: based on arrival
     b.vj("vj2")("B1", "22:00"_t, "22:00"_t)("C1", "23:00"_t, "23:00"_t);
 
     nt::PT_Data& d = *b.data->pt_data;
-    const auto& stl1 = d.vehicle_journeys[0]->stop_time_list;
-    const auto& stl2 = d.vehicle_journeys[1]->stop_time_list;
+    const auto& stl0 = d.vehicle_journeys[0]->stop_time_list;
+    const auto& stl1 = d.vehicle_journeys[1]->stop_time_list;
+    const auto& stl2 = d.vehicle_journeys[2]->stop_time_list;
```

Comment:
stl2 not used

Reply:
nope, it is used here (L 3112)
https://github.com/CanalTP/navitia/pull/2829/files#diff-d7f228dace402afdab826f03fcda090cR3112

Reply:
Ho yeah, didn't expand the diff... -_-